src/main/scala/QuartzSchedulerExtension.scala

```diff
@@ -36,26 +37,30 @@ class QuartzSchedulerExtension(system: ExtendedActorSystem) extends Extension {
   // todo - use of the circuit breaker to encapsulate quartz failures?
   def schedulerName = "QuartzScheduler~%s".format(system.name)
 
-  protected val config = system.settings.config.getConfig("akka.quartz").root.toConfig
+  private val config = system.settings.config.getConfig("akka.quartz").root.toConfig
```

Comment:
This technically might break builds for others as they could be overriding this `val` with their own `val` definition.

Instead of switching this to `private`, can you resubmit this diff with just changing this from `val` to `def`? That should still work for your use-case and will avoid any potential breaking changes.

Thoughts?

Reply:
Done.